src/notation/view/widgets/editstyle.cpp

```diff
@@ -2578,7 +2578,15 @@ void EditStyle::enableStyleWidget(const StyleId idx, bool enable)
 
 void EditStyle::enableVerticalSpreadClicked(bool checked)
 {
-    disableVerticalSpread->setChecked(!checked);
+    if (checked) {
+        setStyleValue(StyleId::enableVerticalSpread, true);
+        disableVerticalSpread->setChecked(false);
+    } else {
+        if (!disableVerticalSpread->isChecked()) {
+            setStyleValue(StyleId::enableVerticalSpread, false);
```

Comment:
Shouldn't this be `true` as well?
```suggestion
            setStyleValue(StyleId::enableVerticalSpread, true);
```